package.json

```diff
@@ -40,6 +40,9 @@
 		"url": "https://phabricator.wikimedia.org/project/board/5674/"
 	},
 	"homepage": "https://github.com/wmde/new-lexeme-special-page#readme",
+	"engines": {
+		"node": ">=16"
+	},
```

Comment:
I can’t fully remember why this is here… I think at some point I got a warning about compatibility with Node 8 (which I guess is the default version if the `package.json` doesn’t specify engines?), but I can’t reproduce this now.

Reply:
Doesn't hurt, and it is consistent with our `.nvmrc`.